OpenRA.Platforms.Default/OpenAlSoundEngine.cs

```diff
@@ -66,7 +66,7 @@ static string[] QueryDevices(string label, int type)
 			do
 			{
 				var str = Marshal.PtrToStringAuto(next);
-				next += UnicodeEncoding.Default.GetByteCount(str) + 1;
+				next += Encoding.Default.GetByteCount(str) + 1;
```

Comment:
`Encoding.Default` === `UnicodeEncoding.Default`. `Default` is a *static* member of the `Encoding` class. This is referring to the same encoding (the current system code page). `Encoding.Unicode` would be referring to a Unicode encoding.

This is a good reason to simply this name - it is inherently confusing.